scripts/install

```diff
@@ -38,76 +42,76 @@ grep -q -R 'jessie-backports' /etc/apt/sources.list{,.d} || {
 }
 
 # Install dependencies
-ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control \
-  || ynh_die "Unable to install dependencies"
+ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control
```

Comment:
I'm not sure if you have to put all the packages or only the one that is dependent? In this case php-net-ldap3